fml/BUILD.gn

```diff
@@ -109,13 +109,17 @@ source_set("fml") {
   ]
 
   deps = [
-    "//third_party/abseil-cpp/absl/debugging:symbolize",
     "//third_party/dart/runtime:dart_api",
 
     # These need to be in sync with the Fuchsia buildroot.
     "//third_party/icu",
   ]
 
+  if (is_mac || is_linux) {
+    # This abseil dependency is only used by backtrace.cc.
+    deps += [ "//third_party/abseil-cpp/absl/debugging:symbolize" ]
+  }
```

Comment:
Can we still include this for unopt builds? It's helpful to be able to use `fml::Backtrace` sometimes for local builds. Not a big deal either though since we can just attach a debugger too...

Reply:
Added back in `unopt` builds.